tests/test_client_response.py

```diff
@@ -458,6 +458,32 @@ def test_charset_no_charset():
     assert response.charset is None
 
 
+def test_content_disposition_full():
+    response = ClientResponse('get', URL('http://def-cl-resp.org'))
+    response.headers = {'Content-Disposition':
+                        'attachment; filename="archive.tar.gz"; foo=bar'}
+
+    assert 'attachment' == response.content_disposition.value
+    assert 'bar' == response.content_disposition.parameters["foo"]
+    assert 'archive.tar.gz' == response.content_disposition.filename
```

Comment:
Worth to add two more cases:
1. `content_disposition.parameters` is immutable
2. second access to `content_disposition` returns the same result if `ClientResponse.headers` were modified in-between (I'm surprised to see it's not read-only property).

Reply:
Will do